### PR TITLE
add windows app to welcome project

### PR DESCRIPTION
**Pull request by edewit**

### CODE REVIEW

File: global.json [hunk 1]
```diff
@@ -341,6 +341,14 @@
             "repoUrl": "git://github.com/feedhenry-templates/welcome-android.git",
             "githubUrl": "https://github.com/feedhenry-templates/welcome-android.git",
             "repoBranch": "refs/heads/master"
+          },
+          {
+            "id": "welcome_project_windows",
+            "name": "Welcome Project-windows",
+            "type": "client_native_windowsphone8",
```
Comment (edewit) [inline global.json]: I'm not sure about the type, do we need to introduce a `windows universal` type?

**Comment by edewit** [comment]
@wei-lee WDYT?

**Comment by wei-lee** [comment]
@edewit TBH, it doesn't really matter. We don't support building for windows 8 on the platform anyway. The type is really for controlling which icon should be used for the app. `client_native_windowsphone8 ` is fine here.

**Comment by edewit** [comment]
can we merge this or do we need a flag to indicate that you can't build this project?

**Comment by wei-lee** [comment]
@edewit actually you are right. We do need a new app type as the platform can not build this new app template. If we reuse the existing 'client_native_windowsphone8' type, the developers will see the build option.

So we need a new app type. I have created a wiki page [here](https://github.com/fheng/millicore/wiki/Add-a-new-app-type-for-templates) to document the steps required to create the new app type. 

Once this new app type is created, PR #22 probably needs to be updated as well to use the new app type.

@jasonmadigan  could you review the wiki as well and see if I missed anything.

**Comment by edewit** [comment]
updated to new type introduced in https://github.com/fheng/millicore/pull/507 and https://github.com/fheng/fh-ngui/pull/538

**Comment by edewit** [comment]
@jasonmadigan can this be merged?

**Comment by jasonmadigan** [comment]
@edewit this looks fine, lets merge.

**Pull request merged**